packages/x-components/src/components/panels/base-tabs-panel.vue

```diff
@@ -0,0 +1,331 @@
+<template>
+  <section v-if="getTabs().length > 0" class="x-tabs-panel" data-test="base-tabs-panel">
+    <component
+      :is="animation"
+      tag="ul"
+      class="x-list"
+      :class="tabsListClass"
+      data-test="base-tabs-panel-list"
+    >
+      <li v-for="tab in getTabs()" :key="tab">
+        <!--
+          @slot Slot that is be used for replacing the whole tab.
+            @binding {tab} string - The tab name.
+            @binding {isSelected} boolean - Indicates if the tab is selected.
+            @binding {selectTab} {() => void} selectTab - Function to select the tab.
+         -->
+        <slot
+          name="tab"
+          v-bind="{ tab, isSelected: tabIsSelected(tab), selectTab: () => selectTab(tab) }"
```

Comment:
In the same way `isSelected` is a simplified name from `tabIsSelected` I'd say we can simplify `selectTab`:
```suggestion
          v-bind="{ tab, isSelected: tabIsSelected(tab), select: () => selectTab(tab) }"
```

Reply:
Sure, good point!